requirements.txt

```diff
@@ -1,13 +1,13 @@
-Flask==1.1.1
-Jinja2==2.11.3
-MarkupSafe==1.1.1
-itsdangerous==1.1.0
-Werkzeug==1.0.1
-requests==2.26.0
-pycryptodome==3.9.6
-cryptography==3.3.2
-responses==0.20.0
-pylint==2.13.3
-pytest==7.1.1
-pytest-cov==3.0.0
-wheel==0.38.1
+Flask==2.2.3
```

Comment:
The `qradar-app-base` image still uses Flask v1, I wonder should these dependencies mirror the base image?

Reply:
Does that matter? Realistically, these requirements are only for build/test, not for base image install and execution. If we stick with older package versions in this project then we're going to hit more Dependabot issues, e.g. cryptography was 3.3.2 and we need 39.0.1 to fix a critical vulnerability.

Reply:
Yep good point, these dependencies are only for the build pipeline so don't need to match.
Looks good!